### PR TITLE
Fix auth0 logout

### PR DESCRIPTION
**Pull request by trevorgerhardt**
This will hopefully fix the ambiguous log out error we're having on production. 😕

### CODE REVIEW

File: lib/reducers/map-state.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const reducers = {
     }
   },
   'log out' (state, action) {
-    return { state: null }
+    return createInitialState()
```
Comment (mattwigway) [inline lib/reducers/map-state.js]: Why does this change mapState?

Comment (trevorgerhardt) [inline lib/reducers/map-state.js]: Application requires "center" so this was causing a React propType error on logout.

Comment (trevorgerhardt) [inline lib/reducers/map-state.js]: The Application component I mean.

Comment (mattwigway) [inline lib/reducers/map-state.js]: Gotcha

**Pull request merged**